parsons/utilities/files.py

```diff
@@ -152,12 +159,21 @@ def zip_check(file_path, compression_type):
         return False
 
 
-def extract_file_name(file_path=None):
+def extract_file_name(file_path=None, include_suffix=True):
     """
-    Extract the file name without the suffix from a file path string.
+    Extract the file name with the file path string.
+
+    file_path: str
+        The file path
+    include_suffix: boolean
+        If True, includes full file name with suffix. If False returns the
+        file name without the suffix (e.g. "myfile.zip" vs. "myfile").
     """
 
     if not file_path:
         return None
 
+    if include_suffix:
+        return file_path.split('/')[-1]
+
     return file_path.split('/')[-1].split('.')[0]
```

Comment:
I know this isn't technically in the scope of the PR, but this line won't properly handle filenames with multiple '.'s (eg `one.two.csv`). instead of just grabbing the first element in the list when splitting on '.', this should join all elements of the list except the last one.

Reply:
Good catch. How would you suggest that I address?